planning/behavior_path_planner/config/avoidance/avoidance.param.yaml

```diff
@@ -16,6 +16,9 @@
       enable_safety_check: false
       enable_yield_maneuver: false
 
+      # bound clipping for objects
+      enable_bound_clipping: false
+
```

Comment:
This can be removed (the parameter is already defined l. 12).
This seems to have been introduced by https://github.com/autowarefoundation/autoware.universe/commit/026a4b40a29d4e96a8d3792f0575efe08f25798a

Reply:
Thank you for pointing it out!
I'll check main branch and fix it

Reply:
```suggestion
```